ui/.env

```diff
@@ -1,3 +1,4 @@
 REACT_APP_AZURE_CLIENT_ID=db8dc4b0-202e-450c-b38d-7396ad9631a5
 REACT_APP_AZURE_TENANT_ID=common
-REACT_APP_API_ENDPOINT=https://feathr-sql-registry.azurewebsites.net
+REACT_APP_API_ENDPOINT=https://feathr-rbac-registry.azurewebsites.net
```

Comment:
>REACT_APP_API_ENDPOINT [](http://example.com/codeflow?start=0&length=22)

Please revert the REACT_APP_API_ENDPOINT default value change prior to PR merge.

Reply:
Reverted